elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -131,6 +132,21 @@ default int getPort() {
     default String getModelPackageName() {
         return "com.yourcompany.elide.models";
     }
+    
+    /**
+     * Package name containing the Async models to support the Async query feature. This package will be 
+     * recursively scanned for @Entity's and registered with Elide.
+     *
+     * NOTE: This will scan for all entities in that package and bind this data to a set named "elideAllModels".
+     *       If providing a custom ElideSettings object, you can inject this data into your class by using:
+     *
+     *       <strong>@Inject @Named("elideAllModels") Set&lt;Class&gt; entities;</strong>
+     *
+     * @return
+     */
+    default String getAsyncModelPackageName() {
```

Comment:
We don't need a setting for this.  The package name is fixed (by us).

Reply:
Done